engine/schema/src/main/resources/META-INF/db/schema-41600to41610.sql

```diff
@@ -20,3 +20,13 @@
 --;
 
 ALTER TABLE `cloud`.`vm_work_job` ADD COLUMN `secondary_object` char(100) COMMENT 'any additional item that must be checked during queueing' AFTER `vm_instance_id`;
+
+-- Add support for VMware 7.0.2.0
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities` (uuid, hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) values (UUID(), 'VMware', '7.0.2.0', 1024, 0, 59, 64, 1, 1);
+-- Copy VMware 7.0.1.0 hypervisor guest OS mappings to VMware 7.0.2.0
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) SELECT UUID(),'VMware', '7.0.2.0', guest_os_name, guest_os_id, utc_timestamp(), 0  FROM `cloud`.`guest_os_hypervisor` WHERE hypervisor_type='VMware' AND hypervisor_version='7.0.1.0';
+
+-- Add support for VMware 7.0.3.0
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities` (uuid, hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) values (UUID(), 'VMware', '7.0.3.0', 1024, 0, 59, 64, 1, 1);
+-- Copy VMware 7.0.1.0 hypervisor guest OS mappings to VMware 7.0.3.0
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) SELECT UUID(),'VMware', '7.0.3.0', guest_os_name, guest_os_id, utc_timestamp(), 0  FROM `cloud`.`guest_os_hypervisor` WHERE hypervisor_type='VMware' AND hypervisor_version='7.0.1.0';
```

Comment:
@weizhouapache check/add any new guest OSes supported for 7.0u2 and 7.0u3.

Reply:
@sureshanaparti 
yes, I have checked theVirtualMachineGuestOsIdentifier of these 3 versions. They are same

https://developer.vmware.com/apis/1192/vsphere

Reply:
# 7.0.3
$ wget https://vdc-download.vmware.com/vmwb-repository/dcr-public/bf660c0a-f060-46e8-a94d-4b5e6ffc77ad/208bc706-e281-49b6-a0ce-b402ec19ef82/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html

# 7.0.2
$ wget https://vdc-download.vmware.com/vmwb-repository/dcr-public/8946c1b6-2861-4c12-a45f-f14ae0d3b1b9/a5b8094c-c222-4307-9399-3b606a04af55/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html

# 7.0.1
$ wget https://vdc-download.vmware.com/vmwb-repository/dcr-public/a5f4000f-1ea8-48a9-9221-586adff3c557/7ff50256-2cf2-45ea-aacd-87d231ab1ac7/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html

$ diff -u vim.vm.GuestOsDescriptor.GuestOsIdentifier.html vim.vm.GuestOsDescriptor.GuestOsIdentifier.html.1
$ diff -u vim.vm.GuestOsDescriptor.GuestOsIdentifier.html vim.vm.GuestOsDescriptor.GuestOsIdentifier.html.2
$